WebApp/autoreduce_webapp/reduction_viewer/views.py

```diff
@@ -403,6 +394,26 @@ def run_summary(request, instrument_name=None, run_number=None, run_version=0):
 
     return context_dictionary
 
+def run_started_by(started_by=None):
+    """
+    Returns name of the user or team that submitted an autoreduction run given the user ID
+    :param started_by:
+    :return:
+    """
+    if started_by is not None:
+        if started_by == -1:
+            started_by = "Development team"
+        elif started_by == 0:
+            started_by = "Autoreduction service"
+        elif started_by > 0:
+            try:
+                user_record = User.objects.get(id=started_by)
+                started_by = f"{user_record.first_name} {user_record.last_name}"
+            except ObjectDoesNotExist as exception:
+                LOGGER.error(exception)
```

Comment:
When we catch this exception, can we also set started_by to `None`.
e.g. if you set this to 400 (something > 0 not in the user table) it will just display `started_by: 400`

Reply:
good point. done.